### ISSUE TITLE
Info Docker Versione

### ISSUE THREAD

**Issue by corasaniti** [netbirdio/netbird#1621]
It's strange not having the CLI installed for the netbirdio/netbird:latest Docker image

I also wanted to know how I can Dockerize my app using the netbirdio/netbird:latest Docker image. For example I would like to install InfluxDB or Mysql in the netbirdio/netbird:latest image

Thanks in advance

**Comment by neo773** [comment]
Hi,
Is this a video bounty?

**Comment by mlsmaycon** [comment]
@neo773 no, that's some bug from Algora.

**Comment by neo773** [comment]
@mlsmaycon 
Could I get assigned then?

**Comment by mlsmaycon** [comment]
no, we don't have a bounty for this issue. I've contacted Algora to check this.

**Comment by adasauce** [comment]
@mlsmaycon thanks for providing this example as a sidecar example, it worked great for me.

I looked around the documentation for something like this and didn't find anything, it might be useful to add to the official docs?

**Comment by mlsmaycon** [comment]
hello @adasauce , yes it would be great! If you want to contribute, you can open a pull request in our docs: https://github.com/netbirdio/docs

**Comment by bodiroga** [comment]
Hi @mlsmaycon!

Thanks for the side-car container example provided at https://github.com/netbirdio/netbird/issues/1621#issuecomment-1963580878, it works great!

Anyway, I have a few questions about the Docker client implementation (by the way, I'm using your cloud service, not a self-hosted installation):

- Why does the ```/go/bin/netbird status``` command report nothing? Here you have what the output looks like:

```bash
Error: failed to connect to daemon error: context deadline exceeded
If the daemon is not running please run:
```
- Why can't I ping my netbird devices by their hostnames? From my computer I can ping them with no problem, but from the Docker container is always says: ```ping: bad address '<hostname>'```

Many thanks again for your work!

**Comment by nazarewk** [comment]
Hello @corasaniti,

We're currently reviewing our open issues and would like to verify if this problem still exists in the [latest NetBird version](https://github.com/netbirdio/netbird/releases).

Could you please confirm if the issue is still there?

We may close this issue temporarily if we don't hear back from you within **2 weeks**, but feel free to reopen it with updated information.

Thanks for your contribution to improving the project!

**Comment by mlsmaycon** [comment]
closing issue due to no recent feedback. Feel free to open a new one if the issue persist or reopen if this was a feature request.
